systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -176,4 +178,34 @@ public static void checkCologForUsedVariable(String varName) {
     public static String globalVariableJsonPathBuilder(int containerIndex, String envVar) {
         return "$.spec.containers[" + containerIndex + "].env[?(@.name=='" + envVar + "')].value";
     }
+
+    public static Properties stringToProperties(String str) {
+        Properties result = new Properties();
+        List<String> list = getLinesWithoutCommentsAndEmptyLines(str);
+        for (String line: list) {
+            String[] split = line.split("=");
+            if (split.length == 1) {
+                result.put(split[0], "");
+            } else {
+                result.put(split[0], split[1]);
+            }
```

Comment:
what about?

```java
result.put(split[0], split.length == 1 ? "" : split[1])
```

Reply:
Yeah I think that's better :) thank you, already pushed and ready for merge